classes/webservice/WebserviceSpecificManagementImages.php

```diff
@@ -669,6 +669,7 @@ protected function manageCustomizationImages()
         $normal_image_sizes = ImageType::getImagesTypes($this->imageType);
         if (empty($this->wsObject->urlSegment[2])) {
             $results = Db::getInstance()->executeS('SELECT DISTINCT `id_cart` FROM `'._DB_PREFIX_.'customization`');
+            $ids = [];
```

Comment:
could you just use the "long" array syntax please ?
$ids = array();

Reply:
Updated.